src/Security/LoginAttempt.php

```diff
@@ -36,7 +36,6 @@ class LoginAttempt extends DataObject
     const FAILURE = 'Failure';
 
     private static $db = array(
-        'Email' => 'Varchar(255)', // Remove in 5.0
```

Comment:
I suspect this is a semver break - can you keep it there but remove it from the filter?